package.json

```diff
@@ -134,23 +134,23 @@
     "file-saver": "2.0.2",
     "focus-visible": "5.0.2",
     "gridicons": "3.3.1",
-    "highlight.js": "9.18.0",
+    "highlight.js": "9.18.1",
     "isemail": "3.2.0",
     "js-yaml": "3.13.1",
     "jszip": "3.2.2",
     "lodash": "4.17.15",
     "md5.js": "1.3.5",
     "pretty-bytes": "5.3.0",
-    "promise": "8.0.3",
+    "promise": "8.1.0",
     "prop-types": "15.7.2",
     "randombytes": "2.1.0",
-    "react": "16.12.0",
+    "react": "16.13.0",
     "react-addons-update": "15.6.2",
-    "react-dom": "16.12.0",
+    "react-dom": "16.13.0",
     "react-dropzone": "10.2.1",
-    "react-modal": "3.11.1",
-    "react-overlays": "2.1.0",
-    "react-redux": "7.1.3",
+    "react-modal": "3.11.2",
+    "react-overlays": "3.0.0",
```

Comment:
Not a very helpful explanation of the breaking change here. I have requested guidance in https://github.com/react-bootstrap/react-overlays/issues/783

Reply:
I just tried to test this in this branch and also in `develop` and I didn't see a popup for email tags. We probably either need to fix it or remove the dependency. Could we take it out of this update PR for now @belcherj to give us time to figure that out?

(Unless you are seeing it work - I'm not entirely sure yet what to look for but I think we should be seeing a popup in the tag list if we try and enter something like an email address.)

Reply:
Sounds good, let's remove it and separate or it.

Reply:
Contact from the library author: v3 did not likely break our code.

Reply:
by the way, with the mystery clear it seems fine to include that back in if you want. no problem leaving it out either

Reply:
Added it back and a couple of other updates: https://github.com/Automattic/simplenote-electron/pull/1951/commits/b2e42574841eed7c55aa61692525451cf6be1f3c